export const getUser = async ({})=>{
    try {
        
        return {user:{name:"Phin"}}

    } catch (error) {
        throw error
    }
}